src/history/hs_rec.c

```diff
@@ -370,6 +370,9 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_MULTI *mult
 
         newest_hs = first_globally_visible_upd = min_ts_upd = out_of_order_ts_upd = NULL;
 
+        __wt_update_vector_clear(&out_of_order_ts_updates);
```

Comment:
Do we need to call these two in the [for loop](https://github.com/wiredtiger/wiredtiger/pull/6984/commits/adcbd02dc2d45e73cbfbce579a2040d5056edf14#diff-43fd2cf13452e67cc179f7b306dc3d93b8db75cf5d0cf6d1cce293b1b260e434R331)?

Reply:
Yes, we have to as they may not cleared in each iteration.